aptos-move/aptos-debugger/src/aptos_debugger.rs

```diff
@@ -78,7 +78,10 @@ impl AptosDebugger {
 
         // TODO(Gas): revisit this.
         let resolver = state_view.as_move_resolver();
-        let vm = AptosVM::new(&resolver);
+        let vm = AptosVM::new(
+            &resolver,
+            /*override_is_delayed_field_optimization_capable=*/ Some(false),
```

Comment:
Why do we need to disable aggregators here?

Reply:
This runs outside of Block-STM context